.changeset/funny-planets-fetch.md

```diff
@@ -0,0 +1,8 @@
+---
+"@razorpay/blade": patch
+---
+
+feat(Alert): design revamp
+- `Alert` is updated to match the new designs
+- Bordered variant is now more compact and smaller in size
+- A new `neutral` intent is added (default)
```

Comment:
Would we need any codemods for this change?

Reply:
Added migration guide in https://github.com/razorpay/blade/pull/840/commits/36440c899db8595e9cd3ac4dd9ad9e846d02d9b4 since there aren't much usage right now of alerts.